app/src/androidTest/java/org/mozilla/fenix/ui/robots/AccountSettingsRobot.kt

```diff
@@ -0,0 +1,63 @@
+package org.mozilla.fenix.ui.robots
+
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.assertion.ViewAssertions
+import androidx.test.espresso.matcher.ViewMatchers
+import org.hamcrest.CoreMatchers
+import org.mozilla.fenix.R
+import org.mozilla.fenix.helpers.click
+
+/**
+ * Implementation of Robot Pattern for the URL toolbar.
+ */
+class AccountSettingsRobot {
+    fun verifyBookmarksCheckbox() = assertBookmarksCheckbox()
+
+    fun verifyHistoryCheckbox() = assertHistoryCheckbox()
+
+    fun verifySignOutButton() = assertSignOutButton()
```

Comment:
small nit, i learned recently that the standard kotlin style is to leave one blank line between all functions.  we should probably do that going forward for all tests for consistency